include/openPMD/auxiliary/JSON.hpp

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include "openPMD/config.hpp"
+
+#if openPMD_HAVE_JSON
```

Comment:
see previous comments: `openPMD_HAVE_JSON` guards are not used in the code anymore.